pkg/gpu/nvidia/alpha_plugin.go

```diff
@@ -86,6 +86,7 @@ func (s *pluginServiceV1Alpha) Allocate(ctx context.Context, rqt *pluginapi.Allo
 	// in cuda10 docker ubuntu base images.
 	resp.Envs = make(map[string]string)
 	resp.Envs["LD_LIBRARY_PATH"] = "/usr/local/nvidia/lib:/usr/local/nvidia/lib64"
+	resp.Envs["PATH"] = "/usr/local/nvidia/bin:/usr/local/cuda/bin"
```

Comment:
You are rewriting the path here, if someone set the path variable in his/her container this will cancel any changes the user made.

Reply:
Right. I realized this issue and added the standard bin paths here. Lets discuss offline on the better long term solution.

Reply:
`resp.Envs["PATH"] = "/usr/local/nvidia/bin:/usr/local/cuda/bin:$PATH"

Reply:
I don't think $PATH would work with container runtime. I think container runtime will just overwrite with the last env it sees. Rethinking about this, it is probably not a good idea to overwrite popular env variables from device plugin. That logic is perhaps still better left to user image and container spec.

Reply:
It will, the Env variables in CRI and OCI is represented as a list, not a map.

You can also get the intuition that it will work because of how this works in Dockerfiles, you can redeclare the PATH variable by using the previous `$PATH` declarations. It works pretty well like an overlay.